src/InteractiveWindow/Version.props

```diff
@@ -1,5 +1,5 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <RoslynSemanticVersion>1.2.0</RoslynSemanticVersion>
+    <RoslynSemanticVersion>2.0.0</RoslynSemanticVersion>
```

Comment:
I see--so it would seem the only *solution* is to move this component to its own repo/component build system (which isn't worth it).  Given this, I approve.